tests/integration/solidity_contracts/PlainOpcodes/test_plain_opcodes.py

```diff
@@ -345,3 +345,44 @@ async def test_should_emit_log3(self, plain_opcodes, addresses, event):
         async def test_should_emit_log4(self, plain_opcodes, addresses, event):
             await plain_opcodes.opcodeLog4(caller_address=addresses[0].starknet_address)
             assert plain_opcodes.events.Log4 == [event]
+
+    class TestCreate2:
+        async def test_should_deploy_bytecode_at_address(
+            self,
+            plain_opcodes,
+            counter,
+            addresses,
+            get_starknet_address,
+            get_solidity_contract,
+        ):
+            salt = 1234
+            evm_address = await plain_opcodes.create2(
+                counter.constructor().data_in_transaction,
```

Comment:
doing an assignment of the bytecode before passing it to the invocation would help the readability of this test, imo. had to scroll back up to the defined method in plaincodes.sol otherwise 
```
bytecode = counter.constructor().data_in_transaction
```

Reply:
make sense when using args like that. I think that `wrap_for_kakarot` is supposed to handle kwargs, I'll check one or the other option

Reply:
I'll use `kwarg` : 

```
            evm_address = await plain_opcodes.create2(
                bytecode=counter.constructor().data_in_transaction,
                salt=salt,
                caller_address=addresses[0].starknet_address,
            )
```

if it's ok to you

Reply:
it is, indeed, okay with me 👍